htdocs/comm/propal/card.php

```diff
@@ -142,6 +142,10 @@
 $usercancreatecontract = ($user->hasRight('contrat', 'creer') == 1);
 $usercancreateintervention = ($user->hasRight('ficheinter', 'creer') == 1);
 $usercancreatepurchaseorder = ($user->hasRight('fournisseur', 'commande', 'creer') || $user->hasRight('supplier_order', 'creer'));
+$usercanreopen = ((empty(getDolGlobalBool('MAIN_USE_ADVANCED_PERMS')) && $usercancreate) || (getDolGlobalBool('MAIN_USE_ADVANCED_PERMS') && !empty($user->hasRight('propal', 'propal_advance', 'reopen'))));
```

Comment:
No empty around a getDolglobalBool, this is already a bool, just do ! to revert the test.